src/macaron/config/defaults.ini

```diff
@@ -47,6 +47,7 @@ recursive = False
 [repofinder]
 find_repos = True
 use_open_source_insights = True
+find_commits = True
```

Comment:
Can you help me understand why we need another config knob here? 🤔 In which case should we not map versions to commits?

Reply:
Given that nothing will be returned if the commit finder fails, it may no longer be an optional feature. So you may be right in that it is not necessary.